UI Responsive

jduong09 opened the pull request:
Resolves #39

jduong09 commented:
## Update 6/7/25
- Removed Dark Mode Button from header
- Added CSS design for landscape mode

jduong09 commented:
## Update
- Fixed design of Pass Badge being outside of card
- Fixed media query incorrect display during desktop mode

ziggysauce reviewed (changes requested):
changes look good. one last thing, it's tough to recreate consistently, but there's some UI bug where when a round ends and certain players have passed, the text overflows onto the other player.

<img width="1209" alt="Screenshot 2025-06-09 at 5 44 57 PM" src="https://github.com/user-attachments/assets/93bc5b4b-4330-4761-bd7d-789408d821ff" />
<img width="1219" alt="Screenshot 2025-06-09 at 5 49 31 PM" src="https://github.com/user-attachments/assets/17dac628-f4b5-4135-aba6-35a6fa8c8323" />

I would say once this gets fixed, this is good to merge in

ziggysauce reviewed (approved):
Changes look good. I'd say this is mergeable. I have another request to improve the UX a bit but I'm gonna make it separate ticket.

jduong09 commented:
## Update 6/10/25
- Removed 'P' and 'W' Pass Badge, and implemented a full word badge on card.
- Removed yellow highlight for winner, and implemented a better designed win state.

ziggysauce reviewed (approved):
ayy sorry for the delay on review -- changes look good! i'm marking as approved since nothing is blocking imo. but i do have one small suggested change below.

simply for consistency, I'd write out "Pass" on the user as opposed to using the "P" badge now that you've changed the UI to show the full word when the other players pass.

<img width="1241" alt="Screenshot 2025-06-24 at 8 39 09 AM" src="https://github.com/user-attachments/assets/4e47dea2-f7fa-48f9-bd17-927390958574" />

jduong09 commented:
Sounds good, updated PR with your comment. 

Merging and closing.
